test/NuGet.Core.Tests/NuGet.PackageManagement.Test/NuGetPackageManagerTests.cs

```diff
@@ -1520,8 +1520,8 @@ public class NuGetPackageManagerTests
 
             Assert.NotNull(exception);
             Assert.True(exception is InvalidOperationException);
-            Assert.Equal("Package 'DoesNotExist' is not found", exception.Message);
-
+            Assert.Contains("Package 'DoesNotExist' is not found", exception.Message);
```

Comment:
Not super important but maybe you also want to verify the correct source is bubbling up. i.e.
Asset.Contains(<YOUR-OFFENDING-SOURCE-NAME>);
(I wonder if you could make the other Assert less dependent on the exact wording - i.e. would Assert.Contains(<YOUR-OFFENDING-PACKAGE_NAME>)
would be just as good a test really...